repository/DoctorRepository.js

```diff
@@ -31,6 +27,31 @@ class DoctorRepository {
         .next();
   }
 
+  // getPatientInfo(patientId) {
+  //   return this.mongo.db('test').collection('user')
+  //       .find({uid: patientId});
```

Comment:
If this is not used, it should be removed

Reply:
You are right. I'll remove it